.taprc

```diff
@@ -0,0 +1 @@
+{"esm": false}
```

Comment:
> Lastly, tap will look for a `tap` section in a `package.json` file in the current working directory, and use that object as as source of configuration as well.
>
> — https://node-tap.org/docs/configuring/

It may be nicer to put this into `package.json` to prevent many tiny dot files.

Reply:
I saw this option and decided against it at the time to keep it very visible that this option was turned off.  Since this is such a weird almost counter-intuitive case, I'll keep it in its own file for now, it that makes sense

Reply:
FWIW, the option name is definitely confusing because the name of the library is confusing (`esm`) but it's generally documented that transpilation of modules needs to be turned off this way when using native modules in tap. So it seems expected, at least right now.

Reply:
Ah gotcha, didn't see this [in your TAP ESM example](https://github.com/jkrems/embracejs/tree/master/examples/tap) and couldn't find this this scenario online - changing now to `package.json`